### PR TITLE
client: fix error handling

### PR DESCRIPTION
**Pull request by safchain**
it is scaring and unuseful to print
stack trace

### CODE REVIEW

File: cmd/client/capture.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ var CaptureCreate = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		client, err := api.NewCrudClientFromConfig(&AuthenticationOpts)
 		if err != nil {
-			logging.GetLogger().Fatalf(err.Error())
+			logging.GetLogger().Critical(err.Error())
```
Comment (nplanel) [inline cmd/client/capture.go]: we should fail (Fatal) here, Critical didn't

Comment (nplanel) [inline cmd/client/capture.go]: or os.Exit(1)

File: cmd/client/capture.go [hunk 2]
```diff
@@ -100,11 +102,12 @@ var CaptureList = &cobra.Command{
 		var captures map[string]api.Capture
 		client, err := api.NewCrudClientFromConfig(&AuthenticationOpts)
 		if err != nil {
-			logging.GetLogger().Fatalf(err.Error())
+			logging.GetLogger().Critical(err.Error())
```
Comment (nplanel) [inline cmd/client/capture.go]: same here

File: cmd/client/capture.go [hunk 3]
```diff
@@ -125,11 +128,12 @@ var CaptureGet = &cobra.Command{
 		var capture api.Capture
 		client, err := api.NewCrudClientFromConfig(&AuthenticationOpts)
 		if err != nil {
-			logging.GetLogger().Fatalf(err.Error())
+			logging.GetLogger().Critical(err.Error())
```
Comment (nplanel) [inline cmd/client/capture.go]: same

File: cmd/client/capture.go (hunk 1 above)
Comment (safchain) [inline cmd/client/capture.go]: yes forgot this one

**Pull request merged**